examples/basic-examples/package.json

```diff
@@ -42,9 +42,6 @@
     "babel-preset-vue": "^2.0.2",
     "core-js": "^2.6.5",
     "cross-env": "^5.2.0",
-    "cypress": "^3.1.5",
-    "cypress-axe": "^0.4.0",
-    "cypress-cucumber-preprocessor": "^1.11.0",
```

Comment:
@dylankelly I think this need to be in our update script so create ripple app can update the project package.json file automatically.